rust/rsc/src/rsc/main.rs

```diff
@@ -52,20 +53,49 @@ struct ServerOptions {
 
     #[arg(help = "Launches the cache without an external database", long)]
     standalone: bool,
+}
 
-    #[arg(
-        help = "Specify an absolute path for a local directory based store",
-        value_name = "LOCAL_STORE",
-        long
-    )]
-    local_store: Option<String>,
+// Maps databse store uuid -> dyn blob::DebugBlobStore
+// that represent said store.
+async fn activate_stores(
```

Comment:
This kind of ties in to the question below. Its not quite just collecting data, it also transforms the database model into the object responsible for doing things. The reason for "activate" was because we had to upgrade the data to a service to make it usable. Otherwise it would've just been called fetch or get.

One thing that may be important to note, this function will also be where other non-local stores are constructed/upgraded in the future